### PR TITLE
Cuahsi branding updates

### PR DESCRIPTION
**Pull request by aphelionz**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case

This is available for viewing at:
https://playground.hydroshare.org/

The changes are as follows:
1. Site title changed from HydroShare to CUAHSI HydroShare
2. "A CUAHSI Water Data Service" added under logo 
3. Meta title changed to "Find, Analyze and Share Water Data"
4. Changed introductory text to "HydroShare is CUAHSI's online collaboration environment for sharing data, models, and code."
5. Changed order of slideshow on home page
6. Added 5th step to "How it works": Formally Publish Data
7. Edited footer content.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by aphelionz** [comment]
retest this, please

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by aphelionz** [comment]
This is available for viewing at:
https://playground.hydroshare.org/

The changes are as follows:
1. Site title changed from HydroShare to CUAHSI HydroShare
2. "A CUAHSI Water Data Service" added under logo 
3. Meta title changed to "Find, Analyze and Share Water Data"
4. Changed introductory text to "HydroShare is CUAHSI's online collaboration environment for sharing data, models, and code."
5. Changed order of slideshow on home page
6. Added 5th step to "How it works": Formally Publish Data
7. Edited footer content.

File: theme/templates/pages/homepage.html [hunk 1]
```diff
@@ -148,6 +147,21 @@ <h4>Share with colleagues</h4>
...
+                    <div class="step-item">
+                        <div class="step-number">5</div>
+                        <h4>Formally Publish</h4>
+                        <div class="description-container table-display">
+                            <span class="step-description table-cell-display">Obtain a Digital Object Identifier so your data can be easily cited.
+                            Reference related journal publications in your metadata.</span>
+                        </div>
+                        <div>
+                            <img class="img-responsive img-step" src="{{ STATIC_URL }}img/home-page/step1.png"
+                                 alt="Create data">
```
Comment (hyi) [inline theme/templates/pages/homepage.html]: Step 5 is using Step1 image and tooltip which does not look right.

File: theme/templates/pages/homepage.html [hunk 2]
```diff
@@ -148,6 +147,21 @@ <h4>Share with colleagues</h4>
                         </div>
```
Comment (hyi) [inline theme/templates/pages/homepage.html]: I think it is better to remove the subject ```You``` on line 142 to make it ```Choose who has access ...``` to be consistent with descriptions for other steps.

**Comment by horsburgh** [comment]
A couple of other things I noticed testing using Chrome on my Mac:

* At some narrower browser widths, the icons under "How it works" are not shrunk symmetrically. Instead they get stretched vertically.

image

* At some narrower browser widths, the text under the navigation bar "HydroShare is CUAHSI's online collaboration environment..." disappears when the navigation items wrap to the next line below the HydroShare logo.

image

**Comment by aphelionz** [comment]
Thanks Jeff! I'll look at that stuff.

**Comment by aphelionz** [comment]
@hyi I've fixed the text changes. I couldn't update the step 5 image because I'm not a graphic designer :(

@horsburgh I fixed some spacing issues but the navbar issue you identified was there before. I changed the display of the "How it works" steps so they change to single column sooner.

**Comment by aphelionz** [comment]
@hyi @dtarb I yanked the logo out for this round of updates, we will revisit that later. For now, everything should be up on playground for review.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by dtarb** [comment]
In testing on playground, I noticed the following quirks in signing up.  I entered USU as my institution.  It showed red.
image

Then when I clicked on the email confirmation I had two institutions in my profile.
image
I am not sure if this is intentional.

**Comment by aphelionz** [comment]
@dtarb Not intentional and likely a separate bug, not an artifact of this PR

**Comment by dtarb** [comment]
On home page I see
image
This is good.

On Discover I see
image
I do not know if you wanted CUAHSI HydroShare here too.

On Collaborate I see Collaborate | CUAHSI HydroShare

On Apps, I get taken to the tethys portal, so am unable to see if branding at the top changed.

If these changes are being made to www, they should probably also be made to help.hydroshare.org.

In the footer I see 
image
This list of projects is shorter than www.  May be a manual edit.

Some of the above I think are known differences between testing and www deployments that you may be able to ignore.  I do not see any show stoppers and am OK with merging this.

+1

**Comment by dtarb** [comment]
Perhaps not quite +1.  The problem with elongated icons that @horsburgh pointed out earlier, still persists.
image
I also think that there should be a different symbol for Formally publish than Create Data.  Same symbol for two things does not seem right.

**Comment by dtarb** [comment]
I also suggest the following wording changes.  Under 4 on how it works.
"Choose who has access to the data and models you have uploaded to HydroShare. You can share with individual users or **make your resources public** for everyone to access."

Under 5.  I suggest "Permanently Publish"  These were words that Olin suggested.  I suggest the detailed text should be: "Obtain a Digital Object Identifier **(DOI)** so your **work** can be easily cited. Reference related journal publications in your metadata."

**Comment by horsburgh** [comment]
Probably something like a larger version of what @Maurier used for the publish icon would be good:

image

**Comment by aphelionz** [comment]
@dtarb @horsburgh Happy to make the changes - do we have any graphic design resources on the team or the team's periphery that we can ask to create that graphic?

**Comment by horsburgh** [comment]
@aphelionz - I think @Maurier did the other icons. @Maurier - do you have a few minutes to create a "publish" icon for the HydroShare front page similar to the one you used on the my resources page (pasted in my comment above)?

**Comment by Maurier** [comment]
You  can use this one:

publish

And here is the .psd file I used to generate them:

[HS ICONS.zip](https://github.com/hydroshare/hydroshare/files/2117352/HS.ICONS.zip)

**Comment by aphelionz** [comment]
@Maurier Thank you!

@dtarb and @horsburgh Updated this. It should be up on playground shortly. Note that the title changes require a local_settings.py fix so that will be a deployment task.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by horsburgh** [comment]
Looks good to me now on playground.

**Comment by dtarb** [comment]
The wording suggestions I made above for changes to 4 and 5 under how it works have not been done.  I am OK if you are deliberately not taking these suggestions.  But if overlooked, please fix.

**Comment by horsburgh** [comment]
@aphelionz - I think @dtarb 's suggested working makes more sense now that there are 5 points instead of 4. The 4th point shouldn't talk about publishing.

**Comment by aphelionz** [comment]
@dtarb @horsburgh All set and should be up on playground shortly.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by dtarb** [comment]
Playground still says "Formally Publish"  It should say "Permanently Publish".  

I did also notice that what was formerly "Discover | HydroShare" has changed to "Discovery | CUAHSI HydroShare".  I think it should be "Discover | CUAHSI HydroShare" to be consistent with the menu item and active rather than passive voice.
image
 
That said - if the switch to Discovery is deliberate motivated by some rationale - I am OK with it.  If it was a slip, then please use "Discover"

**Comment by dtarb** [comment]
@hyi welcome back.  Please review the code, so that when @aphelionz implements the content changes I have commented on this can be merged.

**Comment by hyi** [comment]
@dtarb Will do. I just did a quick check on playground and did find the wording is indeed changed to ```When **permanently** published, this resource will have a formal ...``` unless you meant some other place. I'll defer to @aphelionz whether he would change ```Discovery``` to ```Discover``` as you indicated above. Then I can review the code and approve the PR accordingly.

**Comment by dtarb** [comment]
The word I am asking be changed is
image

**Comment by hyi** [comment]
@dtarb Got it. Thanks. I'll review the code after @aphelionz make these further changes.

**Comment by aphelionz** [comment]
@dtarb @hyi changes have been made and are building on playground now. You may review when ready.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by aphelionz** [comment]
@dtarb Maybe you want to take one last look?

**Comment by dtarb** [comment]
@aphelionz playground is not responding.  Let me know when you want me to take the last look.

**Pull request merged**